v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/templates/SourceDbToSpanner.java

```diff
@@ -130,6 +130,7 @@ static SpannerConfig createSpannerConfig(SourceDbToSpannerOptions options) {
         .withProjectId(ValueProvider.StaticValueProvider.of(options.getProjectId()))
         .withHost(ValueProvider.StaticValueProvider.of(options.getSpannerHost()))
         .withInstanceId(ValueProvider.StaticValueProvider.of(options.getInstanceId()))
-        .withDatabaseId(ValueProvider.StaticValueProvider.of(options.getDatabaseId()));
+        .withDatabaseId(ValueProvider.StaticValueProvider.of(options.getDatabaseId()))
+        .withRpcPriority(ValueProvider.StaticValueProvider.of(options.getSpannerPriority()));
```

Comment:
Can you please verify that all SpannerConfigs throughout the template are created via createSpannerConfig()?

Reply:
yes, I have verified that